src/core/model_config_utils.cc

```diff
@@ -873,6 +868,17 @@ ValidateModelIOConfig(const inference::ModelConfig& config)
 Status
 ValidateBatchIO(const inference::ModelConfig& config)
 {
+  if (
+#ifdef TRITON_ENABLE_TENSORRT
+      (config.platform() != kTensorRTPlanPlatform) &&
+#endif  // TRITON_ENABLE_TENSORRT
+      ((config.batch_input_size() != 0) || (config.batch_output_size() != 0))) {
+    return Status(
+        Status::Code::INVALID_ARG,
+        "batch inputs and batch outputs are only supported for custom "
```

Comment:
Now supported for Tensorflow as well, and soon ORT.

"batch inputs and batch outputs are not support for this platform/backend"

Reply:
Removing. These changes are outdated and unnecessary